tests/src/test/scala/slinky/core/ComponentTest.scala

```diff
@@ -349,9 +347,11 @@ class ComponentTest extends AsyncFunSuite {
   test("Cannot reuse half-built component") {
     val halfBuilt = NoPropsComponent()
     val fullyBuilt: ReactElement = halfBuilt.withKey("abc")
+    println(fullyBuilt)
```

Comment:
Looks like this and the `println` before were added by mistake?

Reply:
No, basically `fullyBuilt` is flagged as unused so we need to either use them somehow or not declare them. But when I don't declare them it seems to fail the test for some reason.
Do you have a better idea than a `println` or something else?

Reply:
Just saw your other comment I'm making it `assert(fullyBuilt != null)`